### ISSUE TITLE
Generator auto-start hours vs minutes mixup

### ISSUE THREAD

**Issue by mpvader** [victronenergy/gui-v2#681]
reported here: https://community.victronenergy.com/questions/249509/venus-os-v32034-available-for-testing.html?page=2&pageSize=50&sort=oldest

![image](https://github.com/victronenergy/gui-v2/assets/5200296/3ce6b51d-683a-4f9c-a155-b710433eb335)

**Comment by chriadam** [comment]
Should be simple fix.  High priority.

**Comment by jpetrell** [comment]
Debugged the UI side (GeneratorStartDialog + Utils.composeDuration()), AFAICS the hours and minutes are correctly turned into seconds that are subsequently passed to VeQuickItem.setValue() that has uid "{serviceUid}/ManualStartTimer".

**Comment by DanielMcInnes** [comment]
This should be testable without a generator, it just opens a relay

**Comment by jpetrell** [comment]
Tested with GFX Ekrano. The hour and minute values are not inverted and I can see that the GUI sets the correct ManualStartTimer value to the system using the dbus-spy, but occasionally something does seem to cause the generator to turn off too early. Setting the timed run to 1 minute lasted 1 minute and setting it to 2 minutes lasted 2 minutes, but when I set the timed run to 1 hour the generator got turned off (=heard the relay snap sound) after roughly 5 minutes. I am not sure what exactly turned off the relay prematurely, but looking at the QML code it is not likely coming from the GUI.
